src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkFileReference.kt

```diff
@@ -191,15 +194,17 @@ class SmkConfigfileReference(
         textRange: TextRange,
         stringLiteralExpression: PyStringLiteralExpression,
         path: String
-) : SmkFileReference(element, textRange, stringLiteralExpression, path) {
-    override fun getVariants() =  collectFileSystemItemLike(
-        onlyFromParentFolder = false,
-        isFromCurrentDir = false
+) : SmkFileReference(
+            element,
+            textRange,
+            stringLiteralExpression,
+            path,
+            searchRelativelyToCurrentFolder = false,
+            makePathRelativelyCurrentFolder = false
     ) {
+    override fun getVariants() =  collectFileSystemItemLike() {
```

Comment:
This line introduces warning that `()` not deeded and better to omit them

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/377/commits/f217b224361103dfa8cba0a8bda0115d475c0bca